simple_use_case/cwl/make_paraview_plot.cwl

```diff
@@ -2,10 +2,20 @@
 
 cwlVersion: v1.0
 class: CommandLineTool
-doc: |
-  Create plot-over-line data with paraview`s pvbatch
-baseCommand: ["pvbatch"]
+doc: Create plot-over-line data with paraview`s pvbatch
+baseCommand: [pvbatch]
 arguments: ["$(inputs.script)", "$(inputs.pvdfile.path)", "plotoverline.csv"]
+
+hints:
+  SoftwareRequirement:
+   packages:
+     paraview:
+       version: [ 5.9.1=hfc1cbd4_3_egl, 5.9.1 ]
+       specs:
+         - https://anaconda.org/conda-forge/paraview
```

Comment:
A special paraview version that runs in batch mode is required (no X-server) to allow paraview to be executed in the github actions, I guess this is the standard version.

Reply:
The test failed, I guess this is due to the missing cwltool installation in the actions file
`
conda install --channel conda-forge cwltool
`